target/linux/layerscape/armv8_64b/target.mk

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (C) 2016 Jiang Yutang <jiangyutang1978@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+ARCH:=aarch64
+BOARDNAME:=ARMv8 64-bit based boards
+CFLAGS:=-Os -pipe -fno-caller-saves
```

Comment:
I just tried to compile without that. There was no problem. Let me remove it in next version. Thanks.